roles/apt-repos/tasks/main.yml

```diff
@@ -3,13 +3,13 @@
   apt_key:
     url: "{{ item.key_url }}"
     validate_certs: "{{ item.validate_certs|default(omit) }}"
-  with_items: repos
+  with_items: "{{ repos|default([]) }}"
```

Comment:
Both Ansible 2.x changes seem like the right direction, but I still feel like a role default makes more sense here and is more "ansibleithic" ?